projects/arlas-components/src/lib/components/mapgl/model/map.type.ts

```diff
@@ -41,10 +41,6 @@ export interface MapOverride {
     position?: 'top-right' | 'top-left' | 'bottom-right' | 'bottom-left',
   ): this;
 
-  removeControl(control: unknown): this;
-
-  hasControl(control: unknown): boolean;
```

Comment:
If we have the possibility to add a control, we should at least provide a way to remove a control